.github/CODEOWNERS

```diff
@@ -7,3 +7,7 @@
 
 # DevOps for Actions and other workflow changes.
 .github/workflows @bitwarden/dept-devops
+
+# Secrets Manager team
+crates/bws @bitwarden/team-secrets-manager-dev
+crates/bitwarden-sm @bitwarden/team-secrets-manager-dev
```

Comment:
Sort alphabetical

Reply:
updated in https://github.com/bitwarden/sdk/pull/494/commits/dfc384108fc41b5d81743f510a6ef49d38e9c183